src/wmnds/components/disruption-indicator/large/_large.njk

```diff
@@ -1,26 +1,26 @@
 {% macro wmndsDisruptionIndicatorLarge(params) %}
 {% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
 
-{% set strongText = params.strongText if params.strongText else "Indicator text" %} {# set button text #}
-{% set iconRight = params.iconRight if params.iconRight %} {# set icon right classes #}
-{% set iconLeft = params.iconLeft if params.iconLeft %} {# set svg left reference #}
-{% set classModifier = ' wmnds-disruption-indicator-large--' + params.classModifier if params.classModifier else "" %}
+{% set _primaryText = params.primaryText if params.primaryText else "Indicator text" %} {# set button text #}
+{% set _iconRight = params.iconRight if params.iconRight %} {# set icon right classes #}
+{% set _iconLeft = params.iconLeft if params.iconLeft %} {# set svg left reference #}
+{% set _classes = ' wmnds-disruption-indicator-large--' + params.classes if params.classes else "" %}
 
 {{params.comment | safe}}
-<div class="wmnds-disruption-indicator-large{{classModifier}}">
+<div class="wmnds-disruption-indicator-large{{_classes}}">
   <div class="wmnds-disruption-indicator-large__left-wrapper">
     <span class="wmnds-disruption-indicator-large__left-icon-wrapper">
       {# Get SVG icon for mode #}
       {{-
         wmndsIcon({
-          icon: iconLeft,
+          icon: _iconLeft,
           class: 'wmnds-disruption-indicator-large__icon'
         }) | indent(2) | trim
```

Comment:
There are a lot of these indents and trims could you delete the ones you find, please..